plugins/processors/execd/execd.go

```diff
@@ -137,6 +144,33 @@ func (e *Execd) cmdReadOut(out io.Reader) {
 	}
 }
 
+func (e *Execd) cmdReadOutStream(out io.Reader) {
+	parser := influx.NewStreamParser(out)
```

Comment:
I assume you need this to avoid side effects!?

Reply:
Yeah, the stream parser reads directly from that io.Reader stream and is able to correctly parse new line characters.